packages/web/src/components/dashboard/supply-overview/SupplyOverview.styles.ts

```diff
@@ -0,0 +1,81 @@
+import styled from "@emotion/styled";
+import mixins from "@styles/mixins";
+import { fonts } from "@constants/font.constant";
+
+export const SupplyOverviewWrapper = styled.div`
+  ${mixins.flexbox("column", "flex-start", "flex-start")}
+  width: 100%;
+  hegiht: 100%;
+  color: ${({ theme }) => theme.color.text02};
+  padding: 24px;
+  gap: 24px;
+  .supply-overview {
+    ${fonts.body9};
+  }
+`;
+
+export const SupplyInfoWrapper = styled.div`
+  ${mixins.flexbox("column", "flex-start", "flex-start")}
+  width: 100%;
+  ${fonts.body7};
+  gap: 16px;
+  .total-supply,
+  .circulating-supply,
+  .daily-block-emissions,
+  .total-staked {
+    ${mixins.flexbox("row", "flex-start", "space-between")};
+    width: 100%;
+  }
+
+  .daily-block-emissions-tooltip {
+    cursor: pointer;
+    ${mixins.flexbox("row", "flex-start", "flex-start")};
+    gap: 4px;
+    svg {
+      width: 18px;
+      height: 18px;
```

Comment:
Missing margin-top here. It's applied to all of the `IconButton`

Reply:
Fixed margins related to IconButton.